### ISSUE TITLE
Full RDP screen after a manual or check command 

### ISSUE THREAD

**Issue by Sebastiangperez** [Fmstrat/winapps#76]
I was able to connect to the vm but i got a full screen remote desktop expreience.
im doing somehting wrong ?

**Comment by Sebastiangperez** [comment]
> I was able to connect to the vm but i got a full screen remote desktop expreience.
> im doing somehting wrong ?

I wanna add that im using Fedora 30 and propietary nvidia drivers ..

**Comment by Fmstrat** [comment]
Do you have `MULTIMON` turned on?

**Comment by Sebastiangperez** [comment]
No, if i turn it on i dont want that the freerdp hangs and can exit.

**Comment by Fmstrat** [comment]
What happens if you hit `Cntrl-Alt-Enter`?

**Comment by Sebastiangperez** [comment]
i just test what you tell me , the windows became small , every time ask me if i want to connect an disconnect from the other machine, and the windows starts to moving.
i have Cinnamon as a desktop enviroment ..

**Comment by Fmstrat** [comment]
Ahhh, it sounds like you're already logged into the machine. Reboot the VM and don't login, then give it another go!

**Comment by Sebastiangperez** [comment]
Ok , i will try later that, thanks 
What about the moving windows when i call some program manually , this has to do with my display driver ? my desktop environment ?

**Comment by Fmstrat** [comment]
Oh I didn't catch that. What do you mean by "moving"? And perhaps that will go away when the login issue is solved.

**Comment by Sebastiangperez** [comment]
i mean , is moving across the screen and shrinks .... like if you are draggin the window.

**Comment by Fmstrat** [comment]
I saw that happen once.. I'm actually really surprised I can't find an issue listed in FreeRDP for it. This may be a new one for upstream.
